examples/tutorials/ProteinLM_Tutorial0.ipynb

```diff
@@ -0,0 +1,2855 @@
+{
```

Comment:
<p>Can we format this portion to be more readable?</p><p>|         | a    | boy  | cute | is   |student| the     | walking | EOS | &nbsp;|---------|------|------|------|------|-------|---------|---------|-----| &nbsp;| a       | 0.125| 0.125| 0.125| 0.125| 0.125 | 0.125   | 0.125   |0.125| &nbsp;| boy     | 0.125| 0.125| 0.125| 0.125| 0.125 | 0.125   | 0.125   |0.125| &nbsp;| cute    | 0.125| 0.125| 0.125| 0.125| 0.125 | 0.125   | 0.125   |0.125| &nbsp;| is      | 0.125| 0.125| 0.125| 0.125| 0.125 | 0.125   | 0.125   |0.125| &nbsp;| student | 0.125| 0.125| 0.125| 0.125| 0.125 | 0.125   | 0.125   |0.125| &nbsp;| the     | 0.125| 0.125| 0.125| 0.125| 0.125 | 0.125   | 0.125   |0.125| &nbsp;| walking | 0.125| 0.125| 0.125| 0.125| 0.125 | 0.125   | 0.125   |0.125| </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4054/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
I have no idea why it looks like that but in Colab and my local Jupyter Notebook it looks like a normal Markdown table?

Reply:
<p>@dhuvik pls shre the code for table you used, I will take a crack at it.</p>

Reply:
<p>wait a second, it looks updated above. Does it reflect for you guys</p>

Reply:
<p>Nope</p>

Reply:
<p>I see the table properly rendered. However, in the paragraph, there is a backslash at the beginning and at the end, can you please remove them: <strong>\ </strong>However, if we look at some of the transition probabilities, we can immediately see that the model is not very good. For example, the probability of the word 'a' coming after 'a' should be close to 0. Same goes for the word 'the' coming after 'a'. It's pretty clear that we need some way of training this model so that we can get some realistic transition probabilities.<strong> \</strong></p>

Reply:
<p>Weirdly enough, when I see the tutorial through NBReviewer, then I see the like Aaron it's not rendering nicely. When I go to discussions, then I see the table perfectly rendered. So I suspect this might be NBReviewer? In the colab I also see it rendered.</p>